scripts/dist-git/mkbranch

```diff
@@ -30,6 +30,7 @@ IGNORE=
 BRANCH=""
 PACKAGES=""
 SRC_BRANCH="master"
+AUTHOR="Fedora Release Engineering <rel-eng@lists.fedoraproject.org>"
```

Comment:
This should be some example@example.com Otherwise once someone start using this package and forget to change this, it sill use this email.